modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -47,8 +47,7 @@ if (cpSku != null) {
 				<div class="availability d-flex mb-4">
 					<div>
 						<commerce-ui:availability-label
-							lowStock="<%= cpContentModel.isLowStock() %>"
-							stockQuantity="<%= cpContentModel.getStockQuantity() %>"
+							namespace="<%= liferayPortletResponse.getNamespace() %>"
```

Comment:
@gianmarcobrunialti we need to pass cpInstanceId and extend CpContentHelper with new method which accepts cpInstanceId, because the tag will not work properly in Catalog

Reply:
this is related to the coment I left, good job in catching this, let's try to refactor all place where we used to have just the request